cms/settings/base.py

```diff
@@ -708,8 +710,8 @@
 # We normally don't want editors to use the images
 # in the rich text editor, for example.
 # They should use the image stream block instead
-RICH_TEXT_BASIC = ["bold", "italic", "link", "ol", "ul", "document-link"]
-RICH_TEXT_FULL = ["h3", "h4", *RICH_TEXT_BASIC]
+RICH_TEXT_BASIC = ["bold", "italic", "link", "ol", "ul"]
+RICH_TEXT_FULL = ["h3", "h4", *RICH_TEXT_BASIC, "document-link"]
```

Comment:
Question: Why are these separate settings rather than using the editors setting below? Saves needing to import them around.

Reply:
We have two types of editors - basic for things like notices, summary etc. And full (for paragraph).

We can swap them around.. ie make basic the default and use the full one where needed